static_src/stores/base_store.js

```diff
@@ -136,6 +136,48 @@ export default class BaseStore extends EventEmitter {
     return cb(true);
   }
 
+  /* mergeAll
+   *
+   * If the dataToMerge exists in the store (as found by
+   * the mergeKey), then merge in new data if there are
+   * changes. The merge algorithm differs from the one
```

Comment:
Just so I understand, the difference between this and mergeMany is this will merge when set of data into mutliple different items in the store?